app/webpacker/components/RegistrationsV2/Register/RegistrationClosedMessage.jsx

```diff
@@ -0,0 +1,48 @@
+import { DateTime } from 'luxon';
+import React, { useEffect, useState } from 'react';
+import { Message } from 'semantic-ui-react';
+import I18nHTMLTranslate from '../../I18nHTMLTranslate';
+
+export default function RegistrationClosedMessage({
+  registrationStart,
+  onTimerEnd,
+}) {
+  const [timeLeft, setTimeLeft] = useState({
+    days: 0, hours: 0, minutes: 0, seconds: 0,
```

Comment:
This should have the correct initial value. Especially since there's a `timeLeft.hours < 1` check below.

Reply:
I call updateTimer immediately in the useEffect. isn't that enough? I haven't seen any rendering issues on local

Reply:
I guess `const [timeLeft, setTimeLeft] = useState(start.diff(DateTime.local(), ['days', 'hours', 'minutes', 'seconds']).toObject());` this would be enough though

Reply:
It's working now, but if someone comes along and modifies the effect, there's a chance of them introducing a bug. For example, any new effects which depend on these values will be triggered on the initial render when they're all 0.
If something can be done without an effect (such as setting this initial value) then it's almost always better to do without.

Reply:
makes sense, I changed it! Will retest on monday